packages/shared/ReactFeatureFlags.js

```diff
@@ -128,3 +128,4 @@ export const deferRenderPhaseUpdateToNextBatch = true;
 
 // Replacement for runWithPriority in React internals.
 export const decoupleUpdatePriorityFromScheduler = false;
+export const enableSetUpdateLanePriority = false;
```

Comment:
I don't think this needs to be a separate flag. Can use `decoupleUpdatePriorityFromScheduler`.

Reply:
Agreed, I thought you said we should use a separate flag so that this one could be hard coded to either true or false while the other one is dynamic but maybe I misunderstood?

Reply:
Either I was wrong or there was a miscommunication :D

One flag is fine